README.rst

```diff
@@ -1,5 +1,38 @@
 HCA DSS CLI
 ===========
+This repository contains a prototype for interacting with the replicated data storage system (aka the "blue box") of the Human Cell Atlas.
+
+This prototype uses a local version of the blue box API spec (using Swagger) to generate a command line client that will make requests to the blue box. Currently, the commands listed are: 
+- upload        Upload a file or directory to the cloud, register each of the files, and bundle them together.
+- download      Download a full bundle or file to local.
+- get-search    Returns a list of bundles matching the given simple criteria.
+- post-search   Accepts Elasticsearch JSON query and returns matching bundle identifiers.
+- get-files     Returns a list of files matching given criteria.
+- head-files    Get the associated metadata from a file with a given uuid.
+- put-files     Create a new file with a given UUID. The file content is passed in through a cloud URL. The file on the cloud provider must have metadata set reflecting the file checksums and the file content type.
+- get-bundles   Returns a list of bundles matching given criteria.
+- put-bundles   Create a new bundle with a given UUID. The list of files UUID+versions to be included must be provided.
+
+Installation
+------------
+To install this package, run :code:`pip install hca-cli`. This will automatically hook up with the api endpoint as defined in the package published last. 
+
+To use the command line interface with a local or test blue box, open <directory_holding_hca_module>/hca/api_spec.json. Change :code:`host` to the host you want (if you're running on a local blue box, this will likely be :code:`localhost:5000`) and the first argument of :code:`schemes` should be the scheme you want (:code:`http` if running locally, :code:`https` otherwise).
+
+Usage
+-----
+The entry to this package is the key word :code:`hca`. Merely typing :code:`hca` will list all commands that are possible in a manner similar to the above. 

```

Comment:
This wording seems a bit awkward, maybe rephrase as "The command-line utility `hca` is the entry point to the CLI provided by this package".